### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
We don't want to recommend not using PageModel.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@danroth27  45 second review.

File: aspnetcore/mvc/razor-pages/index.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ The *Pages/Create.cshtml.cs* code-behind file for the view:
 
 By convention, the `PageModel` class is called `<PageName>Model` and is in the same namespace as the page.
 
-Using a `PageModel` code-behind file supports unit testing, but requires you to write an explicit constructor and class. Pages without `PageModel` code-behind files support runtime compilation, which can be an advantage in development.  <!-- review: advantage because you can make changes and refresh the browser without explicitly compiling the app -->
+Using a `PageModel` code-behind file supports [unit testing](xref:testing/razor-pages-testing), but requires an explicit constructor and class.
```
Comment (danroth27) [inline aspnetcore/mvc/razor-pages/index.md]: I'm fine with the removal of this text, but it feels like this sentence should say a bit more about what a page model is. How about something like: "The `PageModel` class allows you to separate the logic for your page from its presentation. It defines page handlers for request sent to the page and the data used to render the page. This separation allows you to manage page dependencies through dependency inject and to unit test the logic for your page."

**Comment by Rick-Anderson** [comment]
@danroth27  nice.  Minor tweak of your writing + added link to Razor unit testing.

File: aspnetcore/mvc/razor-pages/index.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ The *Pages/Create.cshtml.cs* code-behind file for the view:
 
 By convention, the `PageModel` class is called `<PageName>Model` and is in the same namespace as the page.
 
-Using a `PageModel` code-behind file supports unit testing, but requires you to write an explicit constructor and class. Pages without `PageModel` code-behind files support runtime compilation, which can be an advantage in development.  <!-- review: advantage because you can make changes and refresh the browser without explicitly compiling the app -->
```
Comment (danroth27) [inline aspnetcore/mvc/razor-pages/index.md]: dependency injecting -> dependency injection

**Comment by ohseongkooi** [comment]
(s)

**Pull request merged**